tech/languages/python/multiple-pythons.md

```diff
@@ -26,10 +27,10 @@ Quite a nest, isn't it?
 You can install them like this:
 
 ```bash
-sudo dnf install python33  # to get CPython 3.3
 sudo dnf install python34  # to get CPython 3.4
 sudo dnf install python35  # to get CPython 3.5
 sudo dnf install python36  # to get CPython 3.6
+sudo dnf install python37  # to get CPython 3.7
 sudo dnf install python26  # to get CPython 2.6
```

Comment:
Why Is there `2.6` at the bottom? Maybe put it seperately?

Please also prefix all those with `$`, see our [contributing guide](http://developer-portal.github.io/contributing/write-general-writing-hints.html).

Reply:
It's there, because it's an option.

I'll gladly do that, but it's out of scope of this change.

Reply:
How about a decreasing order, like in the previous list?

Reply:
Whatever you prefer, however, this is a simple addition to the list that follows the current order, however weird it is.

Reply:
You can very well add another commit(s). But if you want to create another PR you may as well. It's up to you.

Reply:
ebc2ea7

Reply:
Added dollars in d01c1d3, note that I fail to use the jekyll beast to test if the console lexer doesn't break anything.

Reply:
Thanks for the work!

> note that I fail to use the jekyll beast to test if the console lexer doesn't break anything.

We'll check that as every merge triggers a developement build to http://developer.fedorainfracloud.org/